Add how-to on Python to MATLAB interface

jwbuurlage opened the pull request.

Commit:
Add how-to on Python to MATLAB interface

ahendriksen commented on how-to-guide.org:
```diff
@@ -9,6 +9,43 @@
 **** PEP8
 **** [[https://github.com/google/yapf][yapf]]
+*** Call MATLAB from Python
+\textsc{Jan-Willem Buurlage}\\
```
Does textsc work for the HTML export?

ahendriksen reviewed (commented):
Looks good! Can you check the textsc?

wjp commented:
Ah, very nice. I quickly tested the instructions, and they work, except that the sample call to `triarea` misses a second argument.

jwbuurlage commented:
I added a second argument, and removed the `\textsc`.

How do you feel about adding section authors? It might motivate people if they get some credits, and it may be useful information as they can initially ask the author for more details if necessary.

ahendriksen commented:
> How do you feel about adding section authors? It might motivate  people if they get some credits, and it may be useful information to  readers as they can initially ask the author for more details if  necessary.

I think it is a good idea for the reasons you cite.

jwbuurlage commented:
Ok, I'll make a separate PR adding authors for every current section.

For me, this how-to is OK to merge.

adriaangraas commented:
Looks all good, authors sounds good. Compile docs?

Commit:
Add authors to completed how-to's

Commit:
Update docs

jwbuurlage commented:
Added everything in this PR: authors + compiled docs.
